### PR TITLE
Added Stanza as Client Side Library

### PR DESCRIPTION
**Pull request by Kalmuraee**

### CODE REVIEW

File: data/libraries.json [hunk 1]
```diff
@@ -803,5 +803,14 @@
...
         "url": "http://ivan.vucica.net"
+    },
+     {
+        "doap": null,
+        "last_renewed": null,
+        "name": "Stanza",
+        "platforms": [
+            "TypeScript"
+        ],
+        "url": "http://stanza,io"
```
Comment (Echolon) [inline data/libraries.json:814]: I think you should replace the , with a .

**Comment by Echolon** [comment]
@Kalmuraee thanks for the PR!

Be reminded that we require project owners to update the timestamp annually (otherwise it will not be shown on the website).

In general I recommend to provide a DOAP file too as we will render clients, server etc. based on these files in the future.

https://xmpp.org/extensions/xep-0453.html

@legastero can you confirm this PR

File: data/libraries.json (hunk 1 above)
Comment (Kalmuraee) [reply data/libraries.json]: Resolved, Seems I'lI need another glasses :)

**Comment by wurstsalat3000** [comment]
Since there is no feedback from the maintainer, I'm going to close this for now. Please re-open if this changes in the future.

**Pull request closed without merging**